src/modules/dashboard/components/DecisionPreview/DecisionPreview.tsx

```diff
@@ -36,11 +38,12 @@ const MSG = defineMessages({
 const decisionData = {
   title: 'Should we build a Discord Bot?',
   description: `I think we should build a Discord bot that integrates with the Dapp and provides our community with greater transperency and also provides more convienience for us to be notified of things happening in our Colony.`,
+  /* Using an HTML string for the dialog content ensures the isDirty prop works as expected */
```

Comment:
How does this isDirty prop work? Is this related to Formik?

Reply:
The prop should be just `dirty`, it is related to Formik. The purpose of it is to determine if the input value has been changed.